wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitUrl.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.git.impl.nativegit;
+package org.eclipse.che.api.git;
```

Comment:
please take a look this class and maybe rework it:
* IMHO GitUrl it would be better to rename it to GitUrlUtils and rewrite java docs.
* getCodenvyTimeStampKeyLabel strange method for me. 'Codenvy' should not be in che project. Try to find better name for it and write java docs. What does it do?